.github/workflows/python-publish.yaml

```diff
@@ -48,3 +48,22 @@ jobs:
         with:
           name: lambda-layer
           path: ./python
+    publish-images:
+      needs: deploy
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v2
+      - name: Log in to container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          tags: ghcr.io/${{ github.repository }}:${{ needs.deploy.outputs.version }}
```

Comment:
I don't think this line is going to work without having set up the `version` output in an earlier step.

See these lines for an example of how this output is generated and set on a variable:
- https://github.com/atlanhq/atlan-java/blob/fc2f7ed0af250de4b72c51ac1572e1e8dede5902/.github/workflows/release.yml#L14-L15
- https://github.com/atlanhq/atlan-java/blob/fc2f7ed0af250de4b72c51ac1572e1e8dede5902/.github/workflows/release.yml#L22-L25

Not sure how we'd get the version from Python (the second bullet) (?)